extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -334,10 +339,12 @@ export class AmpA4A extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
-    this.creativeSize_ = {
-      width: this.element.getAttribute('width'),
-      height: this.element.getAttribute('height'),
-    };
+    this.isFluid = this.element.getAttribute('height') == 'fluid';
```

Comment:
Can rephrase this clause so that you don't need to call this.element.getAttribute('height') twice in the non-fluid case.